src/calls/components/SingleCallsView.jsx

```diff
@@ -134,15 +134,15 @@ class SingleCallsView extends Component {
             decimalSeparator: '.',
             showLabels: true,
             showTitle: true,
-            title: 'Call data',
+            showTitle: false,
             useTextFile: false,
             useBom: true,
-            useKeysAsHeaders: true
+            headers: ['ID poziva', 'Datum', 'Trajanje', 'Osoba', 'Tip', 'Rizik', 'Volonter']
```

Comment:
Instead of hardcoding this, create a mapping from the object keys to their localized strings.